Documentation/articles/content/why_content_pipeline.md

```diff
@@ -0,0 +1,59 @@
+
+> Summing up:
+>
+> **compressed textures** = larger package size (maybe) & lower memory usage & more textures
+
+Now we only looked at iOS, but the same applies to desktop environments, most desktop GPUs support DXT texture compression, so the content pipeline will produce DXT compressed textures which can be loaded and used directly. The only platform that is a pain is Android because the Android platform does not have consistent support for compressed textures at the moment and MonoGame has to decompress DXT textures on the device and use it directly. However, even Android will get compressed texture support eventually. There is also a piece of work happening where the pipeline tool will automatically pick a texture format to use, so for opaque textures, it will use ETC1 (which is supported on all android devices but doesn’t support alpha channels) but for textures with an alpha channel, it will use RGBA4444 (dithered) but also allow the user to pick from a wide variety of compression options manually such as PVRTC, ATITC, DXT/S3TC, ETC1, and RGBA4444. This will give the developer the choice of what to use/support.
+
+## Audio
+
+Now let us look at audio. All the different platforms support different audio formats, if you are handling this yourself you will need to manually convert all your files and include the right ones for each platform. A better option would be to keep one source file (be it .mp3, .wmv, etc) and then convert it to a supported format for the target platform at build time? Ok, it does make for longer build times, but at least we know the music will work. MonoGame uses ffmpeg to do the heavy lifting when converting between formats as it can pretty much convert any type to any other type which is really cool.
+
+Most platforms have audio processing that is optimized to certain compressed formats. By not using them you loose performance and system memory.  Often this is when people resort to just using a format optimized for your current target platform. You could save the sound effects as ADPCM and run pretty optimally on Windows systems. The problem here is only apparent once you try to take your game to another platform where ACPCM is not optimal. You need to manually re-export all your game content in the new optimal format for that platform… and you did keep your original content uncompressed right?
+
+## Shaders
+
+This is an area that causes real pain, custom shaders. There are a number of shading languages that you can use depending on the platform you are targeting. For OpenGL based systems that is GLSL, for DirectX based systems it is HLSL, there is also CG from Nvidia.
```

Comment:
> This is an area that causes real pain, custom shaders.

I am concerned this doc is being too informal with the reader.  They are not looking for jokes or fun... just information.

It is also not pleasant to just load the user down with "this sucks", "that sucks", "this is difficult", etc,  For example... i don't find shaders "a real pain".  They work fine.

Reply:
:D, blame @dellis1972 , likely that was from his article :D 

But I'll do another pass and make it a bit more formal.